src/robot/libdocpkg/xmlwriter.py

```diff
@@ -32,12 +30,11 @@ def write(self, libdoc, outfile):
         self._write_end(writer)
 
     def _write_start(self, libdoc, writer):
-        generated = datetime.utcnow().replace(microsecond=0).isoformat() + 'Z'
         attrs = {'name': libdoc.name,
                  'type': libdoc.type,
                  'format': libdoc.doc_format,
                  'scope': libdoc.scope,
-                 'generated': generated,
+                 'generated': get_timestamp_for_doc(),
```

Comment:
This changes the timestamp from current `2022-05-27T19:08:42Z` to `2022-05-27T19:07:15+00:00`. That ought to be the same thing, and mach `xs:dateTime` that we use in `libdoc.xsd` schema, but because external tools may depend on the exact format this makes the change backwards incompatible. It's not so big change it couldn't be done in RF 5.1, but need to still be mentioned in the release notes. I'll add a label to the issue so that we don't forget about it.